src/app/features/shared/components/molecules/Table/Table.tsx

```diff
@@ -9,7 +9,7 @@ import TableCell from "./components/TableCell/TableCell"
 import TableHeading from "./components/TableHeading/TableHeading"
 import FixedTableCell from "./components/TableCell/FixedTableCell"
 
-type CellContent = string | number | JSX.Element | undefined
+type CellContent = string | number | JSX.Element | undefined | null
```

Comment:
TableCases can possible render `null` since this commit:
57224f2b1e7ad76ab3123cd3a72ebb33581641cb - Non optional Case identification - Gilles Wittenberg - 8/28/20, 11:27 AM

The build complained and failed over it.